Sources/WebSocketKit/WebSocket+Connect.swift

```diff
@@ -17,7 +17,7 @@ extension WebSocket {
         headers: HTTPHeaders = [:],
         configuration: WebSocketClient.Configuration = .init(),
         on eventLoopGroup: EventLoopGroup,
```

Comment:
The function needs @preconcurrency so it emits a warning for non-sendable `onUpgrade`s, instead of throwing errors.